cocos2dx/actions/CCActionEase.h

```diff
@@ -407,7 +407,7 @@ class CC_DLL CCEaseElasticInOut : public CCEaseElastic
 class CC_DLL CCEaseBounce : public CCActionEase
 {
 public:
-    float bounceTime(float time);
+    static float bounceTime(float time);
```

Comment:
why change a member function to static function? This change will break back compatiability.